File: Pratica/beecrowd/javascript/1019.js
const tSegundos = lines[0]

const conversao = (segundos, tipo) => {
    const tempo = {
        hora:3600,
        minuto:60,
        segundo:1
    }
    const quant = Math.floor(segundos/ tempo[tipo])
    const resto = segundos % tempo[tipo]
    return {quant, resto}
} 

function formHora (segundos){
    const horas = conversao(segundos, 'hora')
    const minutos = conversao(horas.resto,'minuto')
    return `${horas.quant}:${minutos.quant}:${minutos.resto}`
}

console.log(formHora(tSegundos))

File: Pratica/beecrowd/java/1019.java
import java.util.Scanner;

public class 1019{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        int horas, minutos, segundos;

        horas = n / 3600;
        minutos = (n%3600) / 60;
        segundos = (n%60);
        
        System.out.println(horas + ":" + minutos +":"+ segundos);
        sc.close();
    }
}